src/microbe_stage/ProcessSystem.cs

```diff
@@ -86,8 +87,15 @@ public ProcessSystem(Node worldRoot)
 
         int hexCount = 0;
 
+        var nucleus = SimulationParameters.Instance.GetOrganelleType("nucleus");
+
         foreach (var organelle in organelles)
         {
+            if (organelle.Definition == nucleus)
+            {
+                hasNucleus = false;
```

Comment:
I don't understand why you take in a parameter to determine if this thing has a nucleus or not, and then you compute the value anyway (but only by **unsetting** the bool from true *if* the thing has a nucleus).